src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
```

Comment:
Don't disable linter

Reply:
It was disabled by default. If I remove it, I have error `A form label must be associated with a control.eslint[jsx-a11y/label-has-associated-control](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md)` near label.

Reply:
Here is a [link](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md) from your error where you can read how to remove this error

Reply:
> It was disabled by default. If I remove it, I have error `A form label must be associated with a control.eslint[jsx-a11y/label-has-associated-control](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md)` near label.

You can add `aria-label` attribute to avoid linter warnings